templates/app/offline-worker.js

```diff
@@ -107,6 +107,15 @@
       });
     },
 
+    // Make requests to directories become requests to index.html
+    extendToIndex: function (request) {
+      var url = request.url;
+      if (url[url.length - 1] === '/') {
+        url += 'index.html';
+      }
+      return new Request(url, request);
```

Comment:
Nit: I guess it doesn't really matter, but we can avoid creating a new Request object when we don't need to rewrite the url.